scaladoc/resources/dotty_res/scripts/inkuire.js

```diff
@@ -0,0 +1,1876 @@
+let WorkerMain;
+(function(){
+'use strict';var d,aa=Object.freeze({assumingES6:!0,productionMode:!0,linkerVersion:"1.5.0",fileLevelThis:this}),l=Math.imul,ba=Math.fround,da=Math.clz32,ea;function fa(a){for(var b in a)return b}function ha(a){this.EK=a}ha.prototype.toString=function(){return String.fromCharCode(this.EK)};var ja=function ia(a,b,c){var f=new a.X(b[c]);if(c<b.length-1){a=a.ws;c+=1;for(var g=f.a,h=0;h<g.length;h++)g[h]=ia(a,b,c)}return f};
```

Comment:
Generated code shouldn't be committed.

Reply:
Fixed, now it is downloaded from Inkuire release assets.

ok, added @BarkingBad and @pikinier20